MotionCompose/app/build.gradle

```diff
@@ -62,21 +62,20 @@ android {
 dependencies {
     implementation 'androidx.core:core-ktx:1.7.0'
     implementation "androidx.compose.ui:ui:$compose_version"
-    implementation 'androidx.compose.material3:material3:1.0.0-alpha02'
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha13'
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
     implementation 'androidx.activity:activity-compose:1.4.0'
-    implementation 'androidx.constraintlayout:constraintlayout-compose:1.0.0-rc02'
+    implementation 'androidx.constraintlayout:constraintlayout-compose:1.0.1'
 
-    def accompanist_version = '0.22.0-rc'
+    def accompanist_version = '0.24.9-beta'
```

Comment:
I believe there is still this release blocking https://github.com/google/accompanist/pull/1186? @bentrengrove do we need to wait for accompanist to upgrade to beta03 of compose?

Reply:
You should match your accompanist version to the compose version so technically yes

Reply:
It probably will be fine though if you aren't using navigation-material

Reply:
v0.24.10-beta is now out

Reply:
Updated to v0.24.10-beta.